spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/loadbalancer/core/RoundRobinLoadBalancer.java

```diff
@@ -62,19 +111,30 @@ public RoundRobinLoadBalancer(String serviceId,
 	// src/main/java/netflix/ocelli/loadbalancer/RoundRobinLoadBalancer.java
 	public Mono<Response<ServiceInstance>> choose(Request request) {
 		// TODO: move supplier to Request?
+		// Temporary conditional logic till deprecated members are removed.
+		if (serviceInstanceListSupplierProvider != null) {
+			ServiceInstanceListSupplier supplier = serviceInstanceListSupplierProvider
+					.getIfAvailable();
+			return Objects.requireNonNull(supplier).get().next()
```

Comment:
I'd rather not have `Objects.requireNonNull()` here. If it's null, it just throws a NPE which isn't helpful. Add a default supplier to `getIfAvailable()` that just returns a noop one.